web/src/main/java/nl/tehdreamteam/se42/web/command/CommandHandler.java

```diff
@@ -0,0 +1,121 @@
+package nl.tehdreamteam.se42.web.command;
+
+import nl.tehdreamteam.se42.web.command.impl.HelpCommand;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Handles {@link Command Commands} registered in this {@code CommandHandler}.
+ */
+public class CommandHandler {
+
+    private static final String COMMAND_ARGUMENT_SPLIT_TOKEN = " ";
+
+    private final Map<String, Command> commands = new HashMap<>();
+
+    public CommandHandler() {
+        register(new HelpCommand());
+    }
+
+    /**
+     * Handles a {@link Command} from the given input-{@code line}.
+     *
+     * @param line The input-{@code line}.
+     */
+    public void handle(String line) {
+        Objects.requireNonNull(line);
+
+        String identifier = parseIdentifierFromLine(line);
+        String input = parseInputFromLine(line);
+        String[] arguments = parseArgumentsFromLine(line);
+        handle(identifier, input, arguments);
+    }
+
+    private String parseIdentifierFromLine(String line) {
+        String[] splits = line.split(COMMAND_ARGUMENT_SPLIT_TOKEN);
+        String identifier = splits[0];
+
+        return identifier;
+    }
+
+    private String parseInputFromLine(String line) {
+        String[] splits = line.split(COMMAND_ARGUMENT_SPLIT_TOKEN);
+
+        String identifier = parseIdentifierFromLine(line);
+        String input = splits.length > 1 ? line.substring(identifier.length() + 1) : "";
+
+        return input;
+    }
+
+    private String[] parseArgumentsFromLine(String line) {
+        String[] splits = line.split(COMMAND_ARGUMENT_SPLIT_TOKEN);
+
+        String[] arguments = Arrays.copyOfRange(splits, 1, splits.length);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "arguments". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1488)